learn/features/governance.md

```diff
@@ -7,15 +7,17 @@ description: 作为波卡平行链，Moonbeam使用链上治理系统来允许
 
 ![Governance Moonbeam Banner](/images/learn/features/governance/governance-overview-banner.png)
 
-## 概览 {: #introduction } 
+## 概览 {: #introduction }
 
 Moonbeam治理机制的目标是根据社区意愿推进协议。在这个共同使命中，治理过程寻求包括所有Token持有者。关于协议的任何以及所有更改都必须通过公投，以便所有的Token持有者可以根据质押权重对决策提出建议。
 
 如[Moonbeam社区论坛](https://forum.moonbeam.foundation/){target=_blank}和[Polkassembly](https://moonbeam.polkassembly.network/){target=_blank}的社区治理论坛将开放讨论，并允许根据社区建议完善提案。自主生效和[无分叉升级](https://wiki.polkadot.network/docs/learn-runtime-upgrades#forkless-upgrades/){target=_blank}将社区团结起来，共同完成推进协议的使命。
```

Comment:
```suggestion
如[Moonbeam社区论坛](https://forum.moonbeam.foundation/){target=_blank}和[Polkassembly](https://moonbeam.polkassembly.io/){target=_blank}的社区治理论坛将开放讨论，并允许根据社区建议完善提案。自主生效和[无分叉升级](https://wiki.polkadot.network/docs/learn-runtime-upgrades#forkless-upgrades/){target=_blank}将社区团结起来，共同完成推进协议的使命。
```

Reply:
I'm going to keep it as `.network` since that is what we use throughout the rest of the docs (english site too) and there doesn't seem to be any redirect to `.io`, both seem to be valid. but if i'm wrong def let me know!

Reply:
ok yeah seems like their docs use `.io` so i'll just switch it 🤷‍♀️